tests/integration_tests/charts/data/api_tests.py

```diff
@@ -476,7 +476,7 @@ def test_chart_data_prophet(self):
         self.assertIn("sum__num__yhat", row)
         self.assertIn("sum__num__yhat_upper", row)
         self.assertIn("sum__num__yhat_lower", row)
-        self.assertEqual(result["rowcount"], 47)
```

Comment:
I'm not sure why (on line #473) `result["data"]` is `[]` for only MySQL.

Reply:
I think we should update the query context so that it's not using the legacy properties (`granularity`, `is_timeseries` etc). I wonder if that may be causing issues.